transposon/submit.sh

```diff
@@ -52,14 +52,21 @@ bclen=$6
 BCreadSeq=$7
 plasmidSeq=$8
 extractBCargs=$9
-
+analyzeBCargs=""
 shift 9
+
+while getopts ':b:' opt; do
+    case $opt in
+    b) analyzeBCargs="$OPTARG";;
+    esac
```

Comment:
shouldn't you catch *?

Reply:
getopts returns `?` when it doesn't recognize the option, thus I've included a case for `?`

Reply:
I would still add a case for * and print something like "ERROR: impossible", this will at least let you know if your logic is failing somehow